data/i18n/en/en.toml

```
@@ -355,6 +355,64 @@ other = "Before you begin"
 [previous_patches]
 other = "Patch Releases:"
 
+[release_binary_alternate_links]
+other  = """You can find links to download Kubernetes components (and their checksums) in the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) files.
+Alternately, use [downloadkubernetes.com](https://www.downloadkubernetes.com/) to filter by version and architecture."""
+
+[release_binary_arch]
+other  = "Architecture"
+
+[release_binary_arch_option]
+other = "Architectures"
+
+[release_binary_copy_link]
+other = "Copy Link"
+
+[release_binary_copy_link_certifcate]
+other = "cert"
+
+[release_binary_copy_link_checksum]
+other = "checksum"
+
+[release_binary_copy_link_signature]
+other = "signature"
+
+[release_binary_download]
+other  = "Download Binary"
+
+[release_binary_options]
+other  = "Download Options"
+
+[release_binary_os]
+other = "Operating System"
+
+[release_binary_os_option]
+other = "Operating Systems"
+
+[release_binary_os_darwin]
+other = "darwin"
+
+[release_binary_os_linux]
+other = "linux"
+
+[release_binary_os_windows]
+other = "windows"
+
+[release_binary_section]
+other = """You can find the links to download the <latest-version> Kubernetes components (along with their checksums) below.
```

```suggestion
other = """You can find the links to download v<current-version> Kubernetes components (along with their checksums) below.
```
because this may eventually appear as part of docs for a release that is _not_ current.

You'll need to change the shortcode to match.

---

Also, please add a TOML comment with a note to localizers that `<current-version>` is a placeholder - some people may not work this out without the hint.